.vscode/extensions.json

```diff
@@ -3,6 +3,7 @@
     "dbaeumer.vscode-eslint",
     "esbenp.prettier-vscode",
     "orta.vscode-jest",
-    "EditorConfig.editorconfig"
+    "editorconfig.editorconfig",
```

Comment:
why different casing?

Reply:
The CamelCase comes from the VS Code Marketplace unique identifier.

I think the lowercase comes from the VS Code option to put installed extensions in the configuration files.

I don't know if the casing is important. We don't want to get a different plugin due to different casing. 🙈

Reply:
in the other file it was uppercase. i think it's case insensitive. so we should lowercase all to be consistent.